### DISCUSSION TITLE
extract_tables: Text in two vertically aligned cells is combined and separated with newline

### DISCUSSION THREAD

**Discussion by dev-guy** [jsvine/pdfplumber#1178] [Ask for help with specific PDFs]
This is a snapshot of a portion of a table that is problematic. It is on a page in landscape format.

Other rows in the table are handled correctly. I am unable to post the PDF for data protection reasons. 

![CleanShot 2024-07-27 at 09 02 01@2x](https://github.com/user-attachments/assets/24e5cbed-66a8-464c-9fcc-52c642fa566e)

`extract_tables()` is called with no arguments. It returns a list that contains the following list:

`['Moisture content\nPolysorbate 80', None, '✓', ], [None, None, 'NT', ]`

The two leftmost cells have been combined into one cell with the words in each cell concatenated and separated by a newline.

A similar issue occurs with the following cells but they appear in the middle of the table: 
![CleanShot 2024-07-29 at 08 33 30@2x](https://github.com/user-attachments/assets/a5b30f80-2140-4d17-b954-7eeadcb9fa94)

I tried changing `snap_tolerance` and `join_tolerance` to no avail. 

I tried changing `text_y_tolerance`. Values >= 17 do help but there are other parts of the table that suffer from the same problem. Increasing `text_y_tolerance` further doesn't fix them. Values much greater than 17 (say 50) cause near-garbage to be returned.

Does anyone know why a horizontal line between cells would be ignored?

Default `extract_tables()` options:

```py
{
    "vertical_strategy": "lines", 
    "horizontal_strategy": "lines",
    "explicit_vertical_lines": [],
    "explicit_horizontal_lines": [],
    "snap_tolerance": 3,
    "join_tolerance": 3,
    "edge_min_length": 3,
    "min_words_vertical": 3,
    "min_words_horizontal": 1,
    "keep_blank_chars": False,
    "text_tolerance": 3,
    "text_x_tolerance": None,
    "text_y_tolerance": None,
    "intersection_tolerance": 3,
    "intersection_x_tolerance": None,
    "intersection_y_tolerance": None,
}
```

**Comment by jsvine** [comment]
Without access to the PDF itself, it may prove difficult to provide suggestions. But as a first step, could you run `page.to_image().debug_tablefinder()` (i.e., with the default settings) and share as much as possible of the resulting iamge?
